### PR TITLE
Updated tutorial.md and metadata.yaml

### PR DESCRIPTION
**Pull request by NicHerndon**
Made language changes and added key points for "Understanding Galaxy history system".

### CODE REVIEW

File: topics/introduction/tutorials/galaxy-intro-history/tutorial.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ annotated.
 All histories begin with the name 'Unnamed history'. Non-anonymous users can rename the history as they see fit:
 
 1. Click the existing name. A text input field will appear with the current name.
-2. Entered a new name or edit the existing the way you'd like.
-3. Press 'Enter' to save the new name. The input field will disappear and the new name display.
+2. Entere a new name or edit the existing one.
```
Comment (bgruening) [inline topics/introduction/tutorials/galaxy-intro-history/tutorial.md]: Enter?

**Comment by bgruening** [comment]
Thanks @NicHerndon!

**Comment by bgruening** [comment]
Oo, now there are merge conflicts. Do you need help to resolve those?

**Comment by NicHerndon** [comment]
Sure.

On Jul 2, 2018, at 4:46 PM, Björn Grüning <notifications@github.com<mailto:notifications@github.com>> wrote:


Oo, now there are merge conflicts. Do you need help to resolve those?

**Comment by bgruening** [comment]
ping @NicHerndon 
Thanks!

**Comment by NicHerndon** [comment]
@bgruening, how should I address these merge conflicts?

**Comment by bgruening** [comment]
Waiting for @shiltemann is usually what I do :) and it seems to have worked here as well :)
Some documentation from github can be found here: https://help.github.com/articles/resolving-a-merge-conflict-on-github/

**Comment by bgruening** [comment]
Thanks @NicHerndon and @shiltemann!

**Pull request merged**